docs/test-start-server.md

```diff
@@ -0,0 +1,44 @@
+#### test-start-server
+---
+Allows test suite to skip the start and stop of the server via pom configuration or command line arguments.
```

Comment:
I decided we didn't really need a test-stop-sever because stop-server can handle an execution call when there isn't a server running